packages/driver/src/baseConn.ts

```diff
@@ -1230,7 +1272,7 @@ export class BaseRawConnection {
         (!inCodec && args !== null) ||
         (this.stateCodec === INVALID_CODEC && state !== Session.defaults())
       ) {
-        [card, inCodec, outCodec] = await this._parse(
+        [card, inCodec, outCodec, _, _, _, warnings] = await this._parse(
```

Comment:
btw, I _think_ these `_` variable names are unnecessary? Maybe eslint would complain if you did this, but I think this is equivalent:

```suggestion
        [card, inCodec, outCodec,,,, warnings] = await this._parse(
```

Just a tiny nit, not even something I'm suggesting we do here, just putting it out there.

Reply:
Huh, I didn't know you could do that 🌈⭐. Though I think I prefer the underscores as it looks more intentional to me, and not that I've just typoed it.